CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
-cmake_minimum_required (VERSION 3.1)
+# Ubuntu 14.04 (Trusty)
+#cmake_minimum_required (VERSION 2.8.12.2)
+# Centos 7
+#cmake_minimum_required (VERSION 2.8.11)
+cmake_minimum_required (VERSION 2.8)
```

Comment:
This should be liftet to 2.8.12 because newer CMakes are deprecating older versions. See build log on GH Actions.

> CMake Deprecation Warning at CMakeLists.txt:5 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

Reply:
This is bumped to 2.8.12